### ISSUE TITLE
dependecy version check fail simplegeneric>0.8

### ISSUE THREAD

**Issue by Xparx** [jupyter/notebook#87]
In a virtualenvironment following the installation instructions after successful install
I try to run

    jupyther notebook

and get the error:

    Traceback (most recent call last):
      File "/home/meuser/.virtualenvs/jupyter-dev/bin/jupyter-notebook", line 4, in <module>
        __import__('pkg_resources').require('jupyter-notebook==4.0.0.dev0')
      File "build/bdist.linux-x86_64/egg/pkg_resources.py", line 2944, in <module>
        def _initialize(g):
      File "build/bdist.linux-x86_64/egg/pkg_resources.py", line 563, in _build_master
        return cls._build_from_requirements(__requires__)
      File "build/bdist.linux-x86_64/egg/pkg_resources.py", line 576, in _build_from_requirements
        dists = ws.resolve(reqs, Environment())
      File "build/bdist.linux-x86_64/egg/pkg_resources.py", line 755, in resolve
        raise DistributionNotFound(req)
    pkg_resources.DistributionNotFound: simplegeneric>0.8

This leads me to belive that simplegeneric is not installed but running:

    pip list

gives

    backports.ssl-match-hostname (3.4.0.2)
    certifi (2015.4.28)
    decorator (3.4.2)
    ipykernel (4.0.0.dev0, /home/meuser/.virtualenvs/jupyter-dev/src/ipykernel)
    ipython (4.0.0.dev0, /home/meuser/.virtualenvs/jupyter-dev/src/ipython)
    ipython-genutils (0.1.0)
    Jinja2 (2.7.3)
    jsonschema (2.4.0)
    jupyter-client (4.0.0.dev0, /home/meuser/.virtualenvs/jupyter-dev/src/jupyter-client)
    jupyter-core (4.0.0.dev0, /home/meuser/.virtualenvs/jupyter-dev/src/jupyter-core)
    jupyter-notebook (4.0.0.dev0, /home/meuser/projects/pyCon_Sweden_2015/jupyter_notebook)
    MarkupSafe (0.23)
    mistune (0.5.1)
    nbconvert (4.0.0.dev0, /home/meuser/.virtualenvs/jupyter-dev/src/nbconvert)
    nbformat (4.0.0.dev0, /home/meuser/.virtualenvs/jupyter-dev/src/nbformat)
    path.py (7.3)
    pexpect (3.3)
    pickleshare (0.5)
    pip (6.1.1)
    ptyprocess (0.4)
    Pygments (2.0.2)
    pyzmq (14.6.0)
    setuptools (8.2)
    simplegeneric (0.8.1)
    terminado (0.5)
    tornado (4.1)
    traitlets (4.0.0.dev0, /home/meuser/.virtualenvs/jupyter-dev/src/traitlets)
    virtualenvwrapper (4.3.1)

I also tried installing simplegeneric 0.8 but without luck.
Any help would be appreciated

**Comment by Carreau** [comment]
I would try to update setuptools, you got 8.2 and the current version is 15 something.

**Comment by Xparx** [comment]
Thanks for the tip for the setuptools, I will look in to that.
This seemed to be some other underlying issue than what I reported, but I think I manage to fix it. 
I noticed that the ipython install suffered the same issue so I reinstalled ipython which made it pick up on the correct version.
I then had to rebuild  and install jupyter again with 

    ./setup.py build
    ./setup.py install

to get past the error though and it seems to have worked.
Sorry for the inconvenience
